src/Components/Address/useAddressAutocomplete.ts

```diff
@@ -74,7 +75,7 @@ export const useAddressAutocomplete = (
       const json = await response.json()
       return json
     },
-    []
+    [apiKey]
```

Comment:
since the ENV should never change i think it is technically ok to leave this out and disable the linter.